test/e2e/devices/devicestoptest/device_stop_test.go

```diff
@@ -12,6 +12,7 @@ import (
 )
 
 func TestCli_Devices_Update(t *testing.T) {
+	helper.IsSkipTest(t)
```

Comment:
These lines and the helper function that they are calling should all be removed.  Specific tests can be run using the [`-run <regexp>` argument for `go test`](https://stackoverflow.com/a/16936314).